### DISCUSSION TITLE
posts/2024/04/field-essay-51/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#49] [General]
# posts/2024/04/field-essay-51/

每年三月份在广德康车二相头人住处打飞，打出三天时间，分为前朝、正朝与尾朝，一天一个村子。其中，正朝一般在二相所处的头人村子中举行，尾朝当天，二相在最后一个村子朝修完毕后需要返回其当期头人住处举行“回拜”。今年茭杯开出的朝修时间依次为三月初九、初十与十一，选中的村庄则依次为马北村、那凡村与得斗村。

https://guozheng.rbind.io/posts/2024/04/field-essay-51/

**Comment by hongtaoh** [comment]
排版真漂亮。以及写的字数够出一本书了

**Reply by residualsun1** [reply]
哇，感谢肯定！欢迎鸿涛来我的小站造访^^（博百家之众长——主题模板在袁凡那儿发现的，字体从益辉那儿拿的。）

内容还需要慢慢打磨，看了一遍还是有错字。不过对我来说，修改博客内容也是对自己的一次次重新认识，也乐在其中（错字除外）。
